Sources/Hub/HubApi.swift

```diff
@@ -213,23 +226,31 @@ public extension HubApi {
 
     /// Performs an HTTP HEAD request to retrieve metadata without downloading content.
     ///
-    /// Allows relative redirects but ignores absolute ones for LFS files.
+    /// Uses a shared URLSession with custom redirect handling that only allows relative redirects
```

Comment:
Ah, the use of a shared session is documented here 👍 I would still mention it when the var is defined.